orm/fields.py

```diff
@@ -93,16 +93,17 @@ def get_column_type(self):
 
 
 class ForeignKey(ModelField, typesystem.Field):
-    def __init__(self, to, allow_null: bool = False):
+    def __init__(self, to, allow_null: bool = False, ondelete: str = None):
```

Comment:
Maybe `on_delete`?

Reply:
Yeah, I also prefer `on_delete`. In this case, I chose `ondelete` to keep consistency with SQLAlchemy's API.

Would you prefer that I rename this to `on_delete`?

Reply:
Yup, I reckon so.